ddl/index.go

```diff
@@ -746,14 +746,16 @@ func (d *ddl) backfillBatchTasks(startTime time.Time, startHandle int64, reorgIn
 		err1 := kv.RunInNewTxn(d.store, true, func(txn kv.Transaction) error {
 			return errors.Trace(reorgInfo.UpdateHandle(txn, nextHandle))
 		})
+		metrics.BatchAddIdxHistogram.WithLabelValues(metrics.BatchAddIndexFail, metrics.RetLabel(err)).Observe(elapsedTime)
```

Comment:
err is not nil, why use metrics.RetLabel(err)?

Reply:
@shenli RetLabel returns "ok" when err == nil and "err" when err != nil, so I can use it.

Reply:
If we can remove `LblResult`.
I think we can put this code on line 743. Then we can remove line 757.